### ISSUE TITLE
Turtlebot4 Simulation turtlebot fails to respond to commands despite /cmv_vel published

### ISSUE THREAD

**Issue by JHPHELAN** [turtlebot/turtlebot4#354]
### Robot Model

applied to both Standard and Lite simulations

### ROS distro

Humble

### Networking Configuration

Not relevant.  Simulation.

### OS

Ubuntu 22.04

### Built from source or installed?

Installed

### Package version

Package: ros-humble-turtlebot4-simulator
Status: install ok installed
Priority: optional
Section: misc
Installed-Size: 41
Maintainer: rkreinin <rkreinin@clearpathrobotics.com>
Architecture: amd64
Version: 1.0.1-1jammy.20240126.090637

### Type of issue

Other

### Expected behaviour

Upon 
"ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py"
or
"ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py slam:=true nav2:=true rviz:=true"
expected Gazebo to launch and robot to respond to on-screen or keyboard movement commands.

### Actual behaviour

The simulated robot failed to respond to movement commands even though 
$ ros2 topic echo /cmd_vel 
demonstrated published commands responding to GUI or keyboard commands.
Sending UNDOCK command via GUI did not solve problem.
Computer reboot and software re-install failed to solve problem.

### Error messages

```bash
NO error messages.
```

### To Reproduce

ubuntu@LinuxBox:~ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py
In Gazebo select BUTTONS.  Click buttons.  No response.
Select KEYBOARD.  Press keys as instructed. No response.
Select Turtlebot4 HMI button (4) [undock].  Try commands above again.  No response.

### Other notes

"ros2 topic info /cmd_vel"
Type: geometry_msgs/msg/Twist
Publisher count: 1
Subscription count: 2

"ros2 topic echo /cmd_vel"
linear:
  x: 1.0
  y: 0.0
  z: 0.0
angular:
  x: 0.0
  y: 0.0
  z: 0.0
---
linear:
  x: 0.0
  y: 0.0
  z: 0.0
angular:
  x: 0.0
  y: 0.0
  z: 0.0
---
linear:
  x: 0.0
  y: 0.0
  z: 0.0
angular:
  x: 0.0
  y: 0.0
  z: -0.5

**Comment by kfu02** [comment]
Note: I have a similar problem, but on hardware. I am publishing messages to /cmd_vel manually from the command line, ros is correctly receiving them (confirmed with an ros2 topic echo), but the turtlebot does not move.

**Comment by RustyCPR** [comment]
@JHPHELAN  Can you please confirm you pressed play?  Located bottom left of the Gazebo screen. 

![ignition](https://github.com/turtlebot/turtlebot4/assets/83372407/27cd0ae8-b411-4f0b-b1ba-0ee59fef9eb9)

**Comment by JHPHELAN** [comment]
Tried both ways.  Play / Pause.  Keyboard / Buttons / Sliders.  No movement.  ros2 topic echo /cmd_vel shows commands are being published.  Just no movement of the virtual robot.

**Comment by RustyCPR** [comment]
HI @JHPHELAN 
Can you please run only: `ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py`
Please share the terminal outputs after you press play and drive.

**Comment by RustyCPR** [comment]
Hi @JHPHELAN - are you still experiencing issues? 
If yes, can you please run only: ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py
Please share the terminal outputs after you press play and drive.
Regards,
Rusty

**Comment by RustyCPR** [comment]
Closing due to inactivity.

**Comment by JHPHELAN** [comment]
I thought I had responded twice with this info, but guess it didn't go through.
[Turtlebot4Errors.txt](https://github.com/turtlebot/turtlebot4/files/14889239/Turtlebot4Errors.txt)
Attached is output of 
ubuntu@linuxbox:~$ ros2 launch turtlebot4_ignition_bringup turtlebot4_ignition.launch.py
When it became clear it wasn't going anywhere, was stopped with ^C.
Additional shutdown output follows.
In this case Gazebo didn't even open but kept requesting world names.
Followed directions from 
[https://turtlebot.github.io/turtlebot4-user-manual/software/turtlebot4_simulator.html](url)
Into a FRESH INSTALL of Ubuntu 22.04 and ROS2 Humble.
Did not install -Common nor -Robot prior as I don't have a physical Turtlebot.
I tried following down the rabbit hole of the deprecation warning and changed fuel.ignitionrobotics.org to fuel.gazebosim.org where I could find them, but that didn't help, so backed out.
Suggestions?  Link to upgraded, non-depricated code?
Thanks!
JHP
